src/main/java/uk/gov/register/thymeleaf/ThymeleafView.java

```diff
@@ -21,17 +17,21 @@
 public class ThymeleafView extends View {
     protected final RequestContext requestContext;
     private final RegisterResolver registerResolver;
-    private final RegisterReadOnly register;
-    private Optional<String> registerTrackingId;
+    protected final RegisterReadOnly register;
+    private final Optional<String> registerTrackingId;
     private String thymeleafTemplateName;
     protected final MarkdownProcessor markdownProcessor = new MarkdownProcessor();
+    protected int totalRecords;
+    protected final int totalEntries;
 
-    public ThymeleafView(RequestContext requestContext, String templateName, RegisterTrackingConfiguration registerTrackingConfiguration, RegisterResolver registerResolver, RegisterReadOnly register) {
+    public ThymeleafView(final RequestContext requestContext, final String templateName, final RegisterTrackingConfiguration registerTrackingConfiguration, final RegisterResolver registerResolver, final RegisterReadOnly register) {
         super(templateName, StandardCharsets.UTF_8);
         this.requestContext = requestContext;
-        this.registerTrackingId = registerTrackingConfiguration.getRegisterTrackingId();
+        registerTrackingId = registerTrackingConfiguration.getRegisterTrackingId();
         this.registerResolver = registerResolver;
         this.register = register;
+        totalRecords = register.getTotalRecords();
```

Comment:
`totalRecords` and `totalEntries` are very view-specific, in that they are only being used by the preview pages and the homepage. I don't think we should be including them as properties on this base view.